sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -272,6 +302,8 @@ PlayRequest getPlayRequest(PlaySource playSource, List<CommunicationIdentifier>
             playSourceInternal = getPlaySourceInternalFromFileSource((FileSource) playSource);
         } else if (playSource instanceof TextSource) {
             playSourceInternal = getPlaySourceInternalFromTextSource((TextSource) playSource);
+        } else if (playSource instanceof SsmlSource) {
+            playSourceInternal = getPlaySourceInternalFromSsmlSource((SsmlSource) playSource);
```

Comment:
I like this approach to get the `playSourceInternal` variable. We should replicate it for `RecognizeRequest` (line 236) to stop that method from growing.

Reply:
yes, refactored the recognizeRequest methods